lib/experimental/fonts/font-library/README.md

```diff
@@ -0,0 +1,250 @@
+    "slug": "piazzolla",
+    "fontFamily": "Piazzolla, serif",
+    "fontFace": [
+        {
+          "uploadedFile": "request-file-id",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Piazzolla"
+        },
+    ]
+}
+```
+
+### Extensibility
+Font Library is able to manage different font collections. A font collection is simply a list of font families ready to be installed by the user. Extenders can provide multiple typographic collections.
+
+#### Provide a font collection
+To provide a font collection you need to call  `wp_register_font_collection ()` function with your configuration as a parameter.
+
+Adding a font collection example:
+
+```php
+$my_config = array (
+    'id'             => 'my-font-collection',
+    'name'           => 'My Font Collection',
+    'description'    => 'Demo about how to a font collection to your WordPress Font Library.',
+    'data_json_file' => path_join( __DIR__, 'my-font-collection-data.json' ),
+);
+
+if ( function_exists( 'wp_register_font_collection' ) ) {
+    wp_register_font_collection ( $my_config );
+}
+```
+
+The Data JSON file for the collection created should look like this. 
+```json
+{
+    "fontFamilies": [
+        ...
+    ],
+    "categories": [
+        ...
+    ]
+}
+```
+
+Example of a data JSON file providing 2 font families with 2 font faces each and 2 categories:
+
+```json
+{
+  "fontFamilies": [
+    {
+      "name": "Montserrat",
+      "fontFamily": "Montserrat, sans-serif",
+      "slug": "montserrat",
+      "category": "sans-serif",
+      "fontFace": [
+        {
+          "src": "http://fonts.gstatic.com/s/montserrat/v25/JTUHjIg1_i6t8kCHKm4532VJOt5-QNFgpCtr6Ew-Y3tcoqK5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Montserrat"
+        },
+        {
+          "src": "http://fonts.gstatic.com/s/montserrat/v25/JTUFjIg1_i6t8kCHKm459Wx7xQYXK0vOoz6jq6R9aX9-p7K5ILg.ttf",
+          "fontWeight": "400",
+          "fontStyle": "italic",
+          "fontFamily": "Montserrat"
+        },
+      ]
+    },
+    {
+      "name": "Piazzolla",
+      "fontFamily": "Piazzolla, serif",
+      "slug": "piazzolla",
+      "category": "serif",
+      "fontFace": [
+        {
+          "src": "http://fonts.gstatic.com/s/piazzolla/v33/N0b52SlTPu5rIkWIZjVKKtYtfxYqZ4RJBFzFfYUjkSDdlqZgy7LYxnLy1AHfAAy5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Piazzolla"
+        },
+        {
+          "src": "http://fonts.gstatic.com/s/piazzolla/v33/N0b72SlTPu5rIkWIZjVgI-TckS03oGpPETyEJ88Rbvi0_TzOzKcQhZqx3gX9BRy5m5M.ttf",
+          "fontWeight": "400",
+          "fontStyle": "italic",
+          "fontFamily": "Piazzolla"
+        },
+      ]
+    }
+  ],
+  "categories": [
+    "sans-serif",
+    "serif"
+  ]
+}
+```
+
+### Create a plugin to provide my font collection
+
+Creating a plugin to provide a font collection can be as simple as this:
```

Comment:
I would rephrase this to: "Here's an example of a plugin that provides a font collection:"